### ISSUE TITLE
Participant counter shows 0 participants

### ISSUE THREAD

**Issue by T-Kay-Work** [engagelab/talkwall#397]
Participant counter shows 0 participants with 2-3 clients having active sessions

**Comment by jandolonen** [comment]
Yes, this was an odd error. Seems to be in combination with a 'server is busy' message that appeared. After that we had a lot of trouble. Happened about 12:30-12:45 CET. After that we were able to log in as students but our student-users didn't show up in the teacher's view. I had to reopen it as a teacher with a new pin code. Then it worked fine again both from a teacher and student perspective.

**Comment by jandolonen** [comment]
Ok, we were not able to reproduce this the next day/test. Due to busy server and issues with pipeline? I reproduced the same action and we got a 'server busy' message today as well but this time it seemed to work fine...
